src/sap.m/src/sap/m/MessageToast.js

```diff
@@ -374,9 +446,32 @@ sap.ui.define(['jquery.sap.global', './InstanceManager', 'sap/ui/core/Popup'],
 			// validate all settings
 			this._validateSettings(mSettings);
 
+			// if content is a string, set as a text control for backward compatibility
+			if (typeof oContent === "string") {
+				oContent = new sap.m.Text({
+					text: oContent,
+					textAlign: "Center"
+				});
+			}
+
+			// group content into a single container control so the popup instance key can be added later
+			oContentContainer = new sap.m.FlexBox({
```

Comment:
all used modules need to be imported using a require/declare statement. Is FlexBox needed?